synapse/rest/client/v1/login.py

```diff
@@ -253,6 +253,8 @@ async def _do_other_login(
 
         Args:
             login_submission:
+            should_issue_refresh_token: Wheter this login should issue a
```

Comment:
'Wheter' has a typo, but generally I prefer to explicitly what it means if it is True:

```suggestion
            should_issue_refresh_token: True if this login should issue a
```